gcloud-java-core/src/test/java/com/google/cloud/ServiceOptionsTest.java

```diff
@@ -215,6 +219,16 @@ public void testBaseEquals() {
   }
 
   @Test
+  public void testLibraryName() {
+    assertEquals(LIBRARY_NAME, OPTIONS.libraryName());
+  }
+
+  @Test
+  public void testApplicationName() {
```

Comment:
Ah, you are right. It could probably be fixed by providing `META-INF/MANIFEST.MF` as a test resource (which would also make the `testApplicationName` a bit more relevant). Feel free to try or ignore or create a separate issue for it.